deepchecks/checks/methodology/test_dataset_size.py

```diff
@@ -0,0 +1,92 @@
+import typing as t
+from deepchecks import Dataset, SingleDatasetBaseCheck, CheckResult, ConditionResult
+
+
+__all__ = ['TestDatasetSize',]
+
+
+TTestSize = t.TypeVar('TTestSize', bound='TestDatasetSize')
+
+
+class TestDatasetSize(SingleDatasetBaseCheck):
+    """Verify Test dataset size."""
```

Comment:
Should be more detailed - 
Verifies test set size compared to training set size

Reply:
updated